referencia/webpayrest/README.md

```diff
@@ -570,22 +580,154 @@ Content-Type: application/json
+
+Puedes [leer más sobre la anulación en la información del
+producto Webpay](/producto/webpay#anulaciones) para conocer
+más detalles y restricciones.
+
+Para anular una transacción se debe invocar al método `Transaction.refund()`.
+
+#### `Transaction.refund()`
+
+Permite solicitar a Webpay la anulación de una transacción realizada previamente y que se encuentra vigente.
+
+> Los SDKs permiten indicar opcionalmente el código de comercio de la
+> transacción a anular, para soportar la anulación en comercios Webpay Plus
+> Mall. En comercios Webpay Plus, no es necesario especificar el código
+> de comercio pues se usa el indicado en la configuración.
+
+<aside class="notice">
+El método `Transaction.refund()` debe ser invocado siempre indicando el código del comercio que realizó la transacción. En el caso de comercios Webpay Plus Mall, el código debe ser el código de la tienda virtual específica.
+</aside>
+
+```java
+// Este SDK aún no se encuentra disponible
+```
+
+```php
+// Este SDK aún no se encuentra disponible
+```
+
+```csharp
+// Este SDK aún no se encuentra disponible
+```
+
+```ruby
+# Este SDK aún no se encuentra disponible
+```
+
+```python
+# Este SDK aún no se encuentra disponible
+```
+
+```http
+POST /rswebpaytransaction/api/webpay/v1.0/transactions/{token}/refunds
+Tbk-Api-Key-Id: 597055555532
+Tbk-Api-Key-Secret: 579B532A7440BB0C9079DED94D31EA1615BACEB56610332264630D42D0A36B1C
+Content-Type: application/json
+
+{
+  "amount": 1000
+}
+```
+**Parámetros**
+
+Nombre  <br> <i> tipo </i> | Descripción
+------   | -----------
+token  <br> <i> String </i> | Token de la transacción. Largo: 64.
+amount  <br> <i> Decimal </i> | (Opcional) Monto que se desea anular de la transacción. Largo máximo: 10.
+
+**Respuesta**
+
+```java
+// Este SDK aún no se encuentra disponible
+```
+
+```php
+// Este SDK aún no se encuentra disponible
+```
+
+```csharp
+// Este SDK aún no se encuentra disponible
+```
+
+```ruby
+# Este SDK aún no se encuentra disponible
+```
+
+```python
+# Este SDK aún no se encuentra disponible
+```
+
+```http
+200 OK
+Content-Type: application/json
+{
+  "type": "NULLIFY",
+  "authorization_code": "123456",
+  "authorization_date": "2019-03-20T20:18:20Z",
+  "nullified_amount": 1000.00,
+  "balance": 0.00,
+  "response_code": 0
+}
+```
+
+Nombre  <br> <i> tipo </i> | Descripción
+------   | -----------
+type  <br> <i> String </i> | Tipo de reembolso (REVERSE. NULLIFY). Largo máximo: 10
+authorization_code  <br> <i> String </i> | Código de autorización de la anulación. Largo máximo: 6
+authorization_date  <br> <i> String </i> | Fecha y hora de la autorización.
+balance  <br> <i> Decimal </i> | Saldo actualizado de la transacción (considera la venta menos el monto anulado). Largo máximo: 17
+nullified_amount  <br> <i> Decimal </i> | Monto anulado. Largo máximo: 17
+response_code <br> <i> Number </i> | Código de resultado del pago. Si es exitoso es 0,de lo contrario el pago no fue realizado. Largo Máximo: 2
```

Comment:
This should be: "Código de resultado de la Anulación/Reembolso"

Reply:
Fixed in d09e83d